config/routes.rb

```diff
@@ -1,56 +1,15 @@
 Rails.application.routes.draw do
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
+  root 'home_page#index'
 
-  # You can have the root of your site routed with "root"
-  # root 'welcome#index'
+  resources :items
 
-  # Example of regular route:
-  #   get 'products/:id' => 'catalog#view'
+  resources :categories
 
-  # Example of named route that can be invoked with purchase_url(id: product.id)
-  #   get 'products/:id/purchase' => 'catalog#purchase', as: :purchase
+  get '/cart', to: 'cart#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.